libsolidity/interface/CompilerStack.cpp

```diff
@@ -1104,6 +1104,7 @@ Json const& CompilerStack::storageLayout(Contract const& _contract) const
 	solAssert(m_stackState >= AnalysisSuccessful, "Analysis was not successful.");
 	solAssert(_contract.contract);
 	solUnimplementedAssert(!isExperimentalSolidity());
+	solUnimplementedAssert(!_contract.contract->storageLayoutSpecifier(), "Storage layout not supported for contract with specified layout base.");
```

Comment:
These should be tested too. Despite the name, they're validations not asserts. If we don't test them, we can't be sure it's properly blocked.

Reply:
Added.